Add tpcc workload docs (#17333) (#21361)
ydb-platform/ydb: ydb/docs/en/core/recipes/ydb-cli/benchmarks.md

Initialization is performed by the `init` command:

```bash
{{ ydb-cli }} workload tpcc --path tpcc/10wh init
{{ ydb-cli }} workload tpch --path tpch/s1 init --store=column
{{ ydb-cli }} workload tpcds --path tpcds/s1 init --store=column
{{ ydb-cli }} workload clickbench --path clickbench/hits init --store=column
```

> At this stage, in case of `tpch`, `tpcds` and `clickbench` you can configure the tables to be created:

Reviewer: ```suggestion
At this stage, for `tpch`, `tpcds`, and `clickbench`, you can configure the tables to be created:
```

* Select the type of tables to be used: row, column, external, etc. (parameter `--store`);
* Select the types of columns to be used: some data types from the original benchmarks can be represented by multiple {{ ydb-short-name }} data types. In such cases, it is possible to select a specific one with `--string`, `--datetime`, and `--float-mode` parameters.

You can also specify that tables should be deleted before creation if they already exist using the `--clear` parameter.

For more details, see the description of the commands for each benchmark:

* [tpcc init](../../reference/ydb-cli/workload-tpcc.md#init)
* [tpch init](../../reference/ydb-cli/workload-tpch.md#init)
* [tpcds init](../../reference/ydb-cli/workload-tpcds.md#init)
* [clickbench init](../../reference/ydb-cli/workload-click-bench.md#init)

> ### Data filling

Reviewer 2: ```suggestion
### Loading data into the tables
```

> Filling with data is performed using the `import` command. This command is specific to each benchmark, and its behavior depends on the subcommands. However, there are also parameters common to all benchmarks.

Reviewer 2: ```suggestion
The `import` command is used to load data into the benchmark tables. This command is specific to each benchmark, and its behavior depends on the subcommands. However, there are also parameters common to all benchmarks.
```
